plugins/woocommerce-blocks/tests/e2e/bin/scripts/products.sh

```diff
@@ -30,6 +30,10 @@ product_id=$(wp post list --post_type=product --field=ID --name="Cap" --format=i
 crossell_id=$(wp post list --post_type=product --field=ID --name="Beanie" --format=ids)
 wp post meta update $crossell_id _crosssell_ids "$product_id"
 
+# Set a product out of stock
+simple_product_id=$(wp post list --post_type=product --field=ID --name="T-Shirt with Logo" --format=ids)
```

Comment:
Good call. I decided to rename ALL of the product variables to follow the same convention, basically `PRODUCT_NAME_product_id` in the same way as categories are names at the top of a file.

I decided NOT to use "functional" names (e.g. `out_of_stock_product`) because some of the products are used in multiple places and also the comments are well describing the purpose.

Changes in: https://github.com/woocommerce/woocommerce/pull/43548/commits/e332281136ec4bf09786a5661958746eb9a5fb8a